data/companies.yml

```
@@ -228,6 +228,13 @@ integrations:
     description: "Application layer protection for Istio service mesh"
     details:
       - "Provides complete layer 7 protection for microservices architecture applications."
+  - name: "Slime"
+    logo: "/logos/slime.png"
+    url: "https://github.com/slime-io"
+    description: "Slime extends, supports, manages Istio and helps to use Istio right."
+    details:
+      - "Slime has built a manager-modules architecture that makes it easy to organize and manage various extensions which are focused on enhancing the capabilities and improving the ease of use of Istio."
+      - "Slime is like Slime, it can be any shape you want and you can do anything in it that is good for Istio's ecology."
```

How about something more like:

```suggestion
    description: "Slime is an intelligent service mesh manager for Istio.
    details:
      - "Slime has a modular architecture that makes it easy to organize and manage various extensions which are focused on enhancing the capabilities and improving the ease of use of Istio."
      - "Using Slime, you can define dynamic service management strategies, which are automatically and conveniently converted to Istio/Envoy configuration."
      - "Just like its namesake, Slime can be any shape you want and it is good for Istio's ecology!"
```